raft/raft_test.go

```diff
@@ -1278,6 +1278,12 @@ func TestLeaderElectionWithCheckQuorum(t *testing.T) {
 	b.checkQuorum = true
 	c.checkQuorum = true
 
+	// we can not let system choosing the value of randomizedElectionTimeout
```

Comment:
can we add a func for this? then we can doc the function instead of adding doc in a few places.